distributionviewer/core/static/js/app/components/views/filtered-charts.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export class FilteredCharts extends React.Component {
+  constructor(props) {
+    super(props);
+    this.filteredClass = ' filtered';
+  }
+
+  componentDidMount() {
+    var wrapper = document.getElementsByClassName('global-wrapper')[0];
+    wrapper.className += this.filteredClass;
```

Comment:
`wrapper.classList.add('filtered')` would let us forego the space prefixed string.